### ISSUE TITLE
Ability to delete a record

### ISSUE THREAD

**Issue by frankietaylor** [aardappel/procrastitracker#4]
It might happen that we might have opened a window and have a face to face discussion with others, during which the application will assume that we have worked on that window for so long.

**Comment by aardappel** [comment]
search for "idle" of this page http://strlen.com/procrastitracker/ for an explanation of how the application deals with this situation.

**Comment by Terrox** [comment]
Is there a way to delete records manually? maybe because you don't really need to track a particular software that you use a lot which has many tabs and records a lot of data you don't need. Or is recording private window tab names from a browser (ie Chrome private tab names)

**Comment by aardappel** [comment]
@Terrox : yup, search for `Keyboard shortcuts` in http://strlen.com/procrastitracker/

**Comment by Nayelianne** [comment]
+1 to being able to actually delete (not just hide) entries.
Being able to pause the tracker without having to close it would be great for avoiding the need to delete entries too (specially since we can't select multiple of them).

**Comment by GustJc** [comment]
There is currently a workaround for that.

You can select the item in the tree and use the `Ctrl+C` shortcut to add **negative** minutes to it.
Put something like "-99999". And it should delete the entry.
